python/sqlalchemy/src/yuga_demo_model.py

```diff
@@ -0,0 +1,104 @@
+"""
+Model objects used by the Yugabyte Demo
+Author: George Nowakowski
+Date Created: April 12, 2019
+"""
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, String, Float, ForeignKey
+from sqlalchemy.orm import sessionmaker, relationship
+
+Base = declarative_base()
+
+
+class User(Base):
```

Comment:
Can we move each of these classes into a top level file? So we end up with four top level files:
* users.py
* products.py
* orders.py

Reply:
This was my initial intuition as well.  Being primarily a Java developer, this is the way classes are organized in Java.

The problem here is that there are bidirectional relationships between the objects.  So a User has a list of Orders, and an Order has a User.  I tried putting these objects each in their own file, but I am getting runtime errors.  The class loader tries to load a User, but User references Order, so it tries to load Order.  Order contains a User --- etc.  This becomes a cyclical relationship.

In scanning the internet, I did not find any good solution to this problem.  The recommended approach is to place the classes with dependencies into the same file.  Python isn't as stringent with class organization, so this is a "pythonic" solution to the problem.

Reply:
Got it. Maybe we name the file simply `model.py` then.